src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -6664,7 +6663,12 @@ namespace AirflowNetworkBalanceManager {
         }
 
         for (i = 1; i <= state.dataAirflowNetwork->AirflowNetworkNumOfSurfaces; ++i) {
-            if (i > state.dataAirflowNetwork->AirflowNetworkNumOfSurfaces - state.dataAirflowNetwork->NumOfLinksIntraZone) continue;
+            if (i > state.dataAirflowNetwork->AirflowNetworkNumOfSurfaces - state.dataAirflowNetwork->NumOfLinksIntraZone) {
+                continue;
+            }
+            if (state.dataAirflowNetwork->AirflowNetworkLinkageData(i).element->type() == ComponentType::SCR) {
+                state.dataAirflowNetwork->AirflowNetworkLinkageData(i).control = state.dataAirflowNetwork->MultizoneSurfaceData(i).Factor;
+            }
```

Comment:
Add the control setting for the linkage here, this is the best place for this to be modified.